### ISSUE TITLE
Inference on only RGB videos

### ISSUE THREAD

**Issue by JagpreetJakhar** [dominickrei/pi-vit#8]
Thanks Dominick for sharing the code, I want to test out the model on only rgb input with csv made  as "path_to_video","label". Is there a way to do it without using skeleton data and any other augmentation. Also I see you used timesformer as baseline for NTU, Could you please give the links or weights for the timesformer finetuned on NTU so I can check it out??

Thanks in advance

**Comment by dominickrei** [comment]
Hi @JagpreetJakhar, to reproduce the TimeSformer baseline I suggest you use [their official codebase](https://github.com/facebookresearch/TimeSformer), our codebase does not support the baseline training out of the box.

I have also uploaded our trained weights for baseline TimeSformer trained on the NTU120 cross-subject split to HuggingFace: https://huggingface.co/datasets/dreilly/pi-vit-checkpoints/blob/main/NTU120_timesformer_baseline.pyth

I hope this helps!

**Comment by JagpreetJakhar** [comment]
Thanks a lot for the helpful info, Dominick! I really appreciate you sharing the link to the TimeSformer weights. I’m also planning to evaluate Pi-Vit on the RGB part of the NTU dataset. Could you guide me on what specific changes I should make to the config for that?
Thanks again for your help!

**Comment by dominickrei** [comment]
For evaluating on RGB only, you should be able to use the command shown in [testing instructions](https://github.com/dominickrei/pi-vit?tab=readme-ov-file#testing) from this repository (in your csv, just set the skeleton feature path to an empty string, they wont be used at inference). No config changes are required.

Another option: the pi-vit weights are backwards compatible with the base TimeSformer code. So if you already have the TimeSformer code setup (from [here](https://github.com/facebookresearch/TimeSformer)), you can pass the pi-vit weights to `TEST.CHECKPOINT_FILE_PATH` and it will give you a result identical to this repository.

**Comment by JagpreetJakhar** [comment]
Got it!, Thanks

**Comment by Sfadingz** [comment]
Hello, as you mentioned  "(in your csv, just set the skeleton feature path to an empty string, they wont be used at inference).",  do you mean setting the path_to_video_1_skeleton in the csv to an empty string? I did so, and got errors in __getitem__ method of NTU dataset. When I keep the path_to_video_1_skeleton, I get a low top-1 accuracy of 66.23% on NTU120-CS, with the provided pretrained weights named NTU120_CS_pivit.pyth.
